app/src/main/res/raw/data.json

```diff
@@ -11365,18 +12393,213 @@
+                "level": "intermediate(B1)"
+            }
+        },
+            {
+                "RU": {
+                    "word": "бюстгальтер",
+                    "topics": ["одежда и обувь"]
+                }
+            }
+        ],
+        [{
+            "EN": {
+                "word": "chain",
+                "topics": ["dress"],
+                "level": "intermediate(B1)"
+            }
+        },
+            {
+                "RU": {
+                    "word": "цепь",
+                    "topics": ["одежда и обувь"]
+                }
+            }
+        ],
+        [{
+            "EN": {
+                "word": "cotton",
+                "topics": ["dress"],
+                "level": "intermediate(B1)"
+            }
+        },
+            {
+                "RU": {
+                    "word": "хлопок",
+                    "topics": ["одежда и обувь"]
+                }
+            }
+        ],
+        [{
+            "EN": {
+                "word": "denim",
+                "topics": ["dress"],
+                "level": "intermediate(B1)"
+            }
+        },
+            {
+                "RU": {
+                    "word": "джинса",
+                    "topics": ["одежда и обувь"]
+                }
+            }
+        ],
+        [{
+            "EN": {
+                "word": "diamond",
+                "topics": ["dress"],
+                "level": "intermediate(B1)"
+            }
+        },
+            {
+                "RU": {
+                    "word": "алмаз",
+                    "topics": ["одежда и обувь"]
+                }
+            }
+        ],
+        [{
+            "EN": {
+                "word": "earring",
+                "topics": ["dress"],
+                "level": "intermediate(B1)"
+            }
+        },
+            {
+                "RU": {
+                    "word": "серьга",
+                    "topics": ["одежда и обувь"]
+                }
+            }
+        ],
+        [{
+            "EN": {
+                "word": "fashionable",
+                "topics": ["dress"],
+                "level": "intermediate(B1)"
+            }
+        },
+            {
+                "RU": {
+                    "word": "модный",
+                    "topics": ["одежда и обувь"]
+                }
+            }
+        ],
+        [{
+            "EN": {
+                "word": "fasten",
+                "topics": ["dress"],
+                "level": "intermediate(B1)"
+            }
+        },
+            {
+                "RU": {
+                    "word": "закреплять",
+                    "topics": ["одежда и обувь"]
+                }
+            }
+        ],
+        [{
+            "EN": {
+                "word": "fur",
+                "topics": ["dress"],
+                "level": "intermediate(B1)"
+            }
+        },
+            {
+                "RU": {
+                    "word": "мех",
```

Comment:
Please, check number of words and maybe remove because of vegans

Reply:
Agree, this should be removed and I need to add words (there are enough in the topic but we have only 13 left in this PR)